tests/pc/test_lag_2.py

```diff
@@ -48,6 +53,14 @@ def common_setup_teardown(ptfhost):
     yield ptfhost
 
     ptfhost.file(path=TEST_DIR, state="absent")
+    # NOTE: As test_lag always causes the route_check to fail and route_check
+    # takes more than 3 cycles(15mins) to alert, the testcase in the nightly after
+    # the test_lag will suffer from the monit alert, so let's config reload the
+    # device here to reduce any potential impact.
+    parallel_run_threaded(
+        target_functions=[lambda duthost=_: config_reload(duthost) for _ in duthosts],
```

Comment:
Config reload takes time to complete. Overall test time may be increased. Is config reload the only way to do teardown? Can we use some monit command to reset the route_check service?

If config reload is the only way to do tear down, please use golden config to do reload. You can refer to PR #10608.

Reply:
How about restarting routecheck after the test?

Reply:
The only problem is, `route_check` monit fails at some point when the testcase after `test_lag_2` runs. And reset/restart doesn't actually restore the devices' routes consistency.

Reply:
So it's not false alert?

Reply:
It's not false alert, the lag member add/remove in `test_lag_2`  causes some route inconsistency. The inconsistency will cause the `route_check` fail after 15 minutes, so the later case will suffer from the monit failure. So we better restore the devices here in `test_lag_2`.

Reply:
> Config reload takes time to complete. Overall test time may be increased. Is config reload the only way to do teardown? Can we use some monit command to reset the route_check service?
> 
> If config reload is the only way to do tear down, please use golden config to do reload. You can refer to PR #10608.

Yes, `config reload` takes time, but the `route_check` failure will cause the later case sanity check failure, which also results in either a `config reload` or `reboot`. As it is sooner or later, we'd better to fix the route inconsistency in `test_lag_2`.

Reply:
2nd thought. Have you ever tried restart the monit service using command like `sudo systemctl restart monit`?
Not sure if the `routeCheck` service will report failure again after 15 minutes even if `monit` is restarted.